go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/ElrondNetwork/elrond-go
 go 1.17
 
 require (
-	github.com/ElrondNetwork/elastic-indexer-go v1.3.0
-	github.com/ElrondNetwork/elrond-go-core v1.1.24
-	github.com/ElrondNetwork/elrond-go-crypto v1.2.2-0.20221110121517-4e85e482f854
-	github.com/ElrondNetwork/elrond-go-logger v1.0.9
-	github.com/ElrondNetwork/elrond-go-p2p v1.0.4-0.20221115083446-a589f335a708
-	github.com/ElrondNetwork/elrond-go-storage v1.0.1
-	github.com/ElrondNetwork/elrond-vm-common v1.3.25
-	github.com/ElrondNetwork/wasm-vm-v1_2 v1.2.45-rc
-	github.com/ElrondNetwork/wasm-vm-v1_3 v1.3.45-rc
-	github.com/ElrondNetwork/wasm-vm-v1_4 v1.4.63
+	github.com/ElrondNetwork/elastic-indexer-go v1.3.3
+	github.com/ElrondNetwork/elrond-go-core v1.1.26
+	github.com/ElrondNetwork/elrond-go-crypto v1.2.3-0.20221207130836-796c2dff9ad7
```

Comment:
do not forget about proper release here and on L10

Reply:
:+1:

Reply:
added proper releases